### PR TITLE
Add yarn audit check -> WEB

### LINKED ISSUE #1806: Add yarn(or npm) audit to our build cci
Description:

We may want to add:
```
yarn audit
```

In our checks, and expect always a 0 at least in the critical and high sections, to minimize the number of unsecure packages we use.

### PR DESCRIPTION
**Pull request by sabau**
Closes #1806 

_Description:_

- Prevent usage of unsafe packages
- Is expected to fail until we merge Web PR that contains tendermint and webpack upgrades

- Should be merged after #1804 

❤️ Thank you!

---

- [ ] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer

### CODE REVIEW

**Comment by sabau** [comment]
Need @faboweb PR to upgrade Webpack

File: .circleci/config.yml [hunk 1]
```diff
@@ -66,6 +66,10 @@ jobs:
           command: yarn run test:unit
           no_output_timeout: 120
 
+      - run:
```
Comment (sabau) [inline .circleci/config.yml]: move it to separate check, so on github we'll see this as a separate failure/succeding test

**Comment by faboweb** [comment]
This fails even on the web PR

**Comment by sabau** [comment]
Yeah I opened a [discussion](https://discuss.circleci.com/t/unexpected-exit-codes-after-explicit-exit-1-0/27716) in the forum, but they may take some time to answer.
Do we have support in our cicleci contract? If yes I'll open a proper ticket for them

**Comment by faboweb** [comment]
@ebuchman re support

**Pull request merged**